LICENSE.md

```diff
@@ -1 +1,287 @@
-For licensing details please consult https://github.com/whosonfirst/whosonfirst-data/blob/master/LICENSE.md
+# License
+
+- **Austria Open Data**: [source data](https://www.data.gv.at/katalog/dataset/c33d36b0-f184-4f2a-89cc-839ca7fcf88a), [source license](http://creativecommons.org/licenses/by/3.0/at/), [more info](https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/README.md)
```

Comment:
- Remove the `[more info]` link since we're not deep linking anymore
- At top, add general link to the detailed source list.
- Add intro preamble as in the README here in the LICENSE.md file between `License` line and first source line (Australian Open Data in this case).
- source [data] and [license]
    > **Austria Open Data**: source [data]() and [license]()

Reply:
fixed in fixed 09a748c and 2f31015